source/index.rst

```diff
@@ -7,6 +7,7 @@ ReadTheRecipe
    breakfast/index
    snack/index
    soup/index
+   sides/index
```

Comment:
Not a big deal, but what is the ordering here? Should sides go at the bottom?

Reply:
Sounds good to me. I was not sure if we wanted it more as an appetizers/sides or if it should be at the end.

Reply:
That makes sense. Do we want an "appetizers" category above soups since soups could be their own "dinner" category?

Reply:
I like that idea, appetizers should be separate.

Reply:
Okay, sounds good to me. I'll change it to be snack, appetizers, ... main food ..., sides, dessert. Does that sound good?

Reply:
I put appetizers right after snacks and put sides right before desserts. We can of course adjust this later if we change our minds.